File: C,C++/p12250.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

char *english = "HELLO\n";
char *spanish = "HOLA\n";
char *german = "HALLO\n";
char *french = "BONJOUR\n";
char *italian = "CIAO\n";
char *russian = "ZDRAVSTVUJTE\n";

int main() {
    int i = 1;
    while (1) {
        char *input = (char*) malloc(14*sizeof(char));
        // if (input[0] == '#') break;
        size_t buffersize = 14;
        getline(&input, &buffersize, stdin);

        if (strcmp(input, "#") == 0 || strcmp(input, "#\n") == 0) break;
        
        if (strcmp(input, english) == 0) {
            printf("Case %d: %s\n", i, "ENGLISH");
        } else if (strcmp(input, spanish) == 0) {
            printf("Case %d: %s\n", i, "SPANISH");
        } else if (strcmp(input, german) == 0) {
            printf("Case %d: %s\n", i, "GERMAN");
        } else if (strcmp(input, french) == 0) {
            printf("Case %d: %s\n", i, "FRENCH");
        } else if (strcmp(input, italian) == 0) {
            printf("Case %d: %s\n", i, "ITALIAN");
        } else if (strcmp(input, russian) == 0) {
            printf("Case %d: %s\n", i, "RUSSIAN");
        } else {
            printf("Case %d: UNKNOWN\n", i);
        }

        i++;
    }
    return 0;
}

File: Java/p12250.java

import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author DarKnight98
 */
public class p12250 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int counter = 1;
        while (true) {
            String masukan = input.nextLine();
            if (masukan.charAt(0) == '#') break;
            
            if (masukan.equals("HELLO")) {
                System.out.printf("Case %d: %s\n", counter, "ENGLISH");
            } else if (masukan.equals("HOLA")) {
                System.out.printf("Case %d: %s\n", counter, "SPANISH");
            } else if (masukan.equals("HALLO")) {
                System.out.printf("Case %d: %s\n", counter, "GERMAN");
            } else if (masukan.equals("BONJOUR")) {
                System.out.printf("Case %d: %s\n", counter, "FRENCH");
            } else if (masukan.equals("CIAO")) {
                System.out.printf("Case %d: %s\n", counter, "ITALIAN");
            } else if (masukan.equals("ZDRAVSTVUJTE")) {
                System.out.printf("Case %d: %s\n", counter, "RUSSIAN");
            } else {
                System.out.printf("Case %d: %s\n", counter, "UNKNOWN");
            }
            
            counter++;
        }
    }
}
